src/assets/reset.js

```diff
@@ -223,8 +223,10 @@ export const normalize = css`
 	button:-moz-focusring,
 	[type='button']:-moz-focusring,
 	[type='reset']:-moz-focusring,
-	[type='submit']:-moz-focusring {
-		outline: 1px dotted ButtonText;
+	[type='submit']:-moz-focusring,
+	:focus {
+		outline: 3px solid aqua;
+		outline-offset: 2px;
```

Comment:
Maki9ng changes to this file means we can't updated it anymore now. Might be better to just add an override and keep this file the same?